app/views/companies/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<% content_for(:body_attributes) do %>
+  data-turbolinks="false"
+<% end %>
+
+<section id="event">
+  <div class="container">
+    <h2 class="text-center text-uppercase text-secondary mb-0">Request New Business Listing</h2>
+    <hr class="star-dark mb-5">
+
+    <%= form_for @company, url: { action: "create" } do |form| %>
+      <div class="form-group">
+        <%= form.label :company_name, "Company Name" %>
+        <%= form.text_field :company_name, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= form.label :website, "Website" %>
+        <%= form.text_field :website, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= form.label :image_url, "Url for Company Logo" %>
+        <%= form.text_field :image_url, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= form.label :additional_comments, "Additional Comments (NOTE: this information will be posted to GitHub)" %>
```

Comment:
Maybe change to "publicly posted to GitHub" -- otherwise it might not be clear to a nontechnical person what this warning is supposed to indicate